.github/workflows/test-elixir.yaml

```diff
@@ -0,0 +1,93 @@
+name: test-elixir
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - "prql-elixir/**"
+      - ".github/workflows/test-elixir.yaml"
+  workflow_call:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+defaults:
+  run:
+    working-directory: prql-elixir
+
+env:
+  MIX_ENV: test
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
No problem if this doesn't work on Mac yet — it's even fine to merge if it only works on Linux at the moment.

But let's put a TODO so that doesn't get lost, and possibly include it in the description in the Readme

Reply:
Actually, it should work on mac, but the action to setup elixir doesn't have Mac support yet

Reply:
Added TODO